app/javascript/mastodon/features/emoji/emoji_compressed.js

```diff
@@ -9,7 +9,12 @@ const { unicodeToFilename } = require('./unicode_to_filename');
 const { unicodeToUnifiedName } = require('./unicode_to_unified_name');
 const emojiMap         = require('./emoji_map.json');
 const { emojiIndex } = require('emoji-mart');
-const { default: emojiMartData } = require('emoji-mart/dist/data');
+const { uncompress: emojiMartUncompress } = require('emoji-mart/dist/utils/data');
+let data = require('emoji-mart/data/all.json');
+if(data.compressed) {
+data = emojiMartUncompress(data);
```

Comment:
I'm not sure this whole if is needed. When I was developing I saw the compressed version of the file, but since then I'm only seeing the uncompressed and I don't know if it's some weird caching.